tests/release-build.sh

```diff
@@ -11,7 +11,11 @@ echo 'nodeos from source should perform a "release build" which excludes asserts
 echo 'debugging symbols, and performs compiler optimizations.'
 echo ''
 
-NODEOS_DEBUG=$(programs/nodeos/nodeos --extract-build-info >(python3 -c 'import json,sys; print(str(json.load(sys.stdin)["debug"]).lower());') &> /dev/null)
+TDIR=$(mktemp -d || exit 1)
+NODEOS_DEBUG=$(programs/nodeos/nodeos --config-dir "${TDIR}" --data-dir "${TDIR}" --extract-build-info >(python3 -c 'import json,sys; print(str(json.load(sys.stdin)["debug"]).lower());') &> /dev/null)
+#avoiding an rm -rf out of paranoia, but with the tradeoff this could change somehow in the future
```

Comment:
Probably not a bad thing if it does break in the future, at least it will be obvious why and might actually prevent some accidental add of something.